src/client/Microsoft.Identity.Client/PublicApi/net472/PublicAPI.Shipped.txt

```diff
@@ -417,6 +417,7 @@ Microsoft.Identity.Client.BrokerOptions.MsaPassthrough.set -> void
 Microsoft.Identity.Client.BrokerOptions.OperatingSystems
 Microsoft.Identity.Client.BrokerOptions.OperatingSystems.None = 0 -> Microsoft.Identity.Client.BrokerOptions.OperatingSystems
 Microsoft.Identity.Client.BrokerOptions.OperatingSystems.Windows = 1 -> Microsoft.Identity.Client.BrokerOptions.OperatingSystems
+Microsoft.Identity.Client.BrokerOptions.OperatingSystems.OSX = 2 -> Microsoft.Identity.Client.BrokerOptions.OperatingSystems
```

Comment:
These need to go to "Unshipped.txt" and we will move them to "Shipped.txt" when we release.